src/web/routes/application/card-address/card-address.js

```diff
@@ -0,0 +1,21 @@
+const pageContent = ({ translate }) => ({
+  title: translate('cardAddress.title'),
+  heading: translate('cardAddress.heading'),
+  buildingAndStreetLabel: translate('cardAddress.buildingAndStreetLabel'),
+  line1Of2: translate('cardAddress.line1Of2'),
+  buildingAndStreetLine2of2: translate('cardAddress.buildingAndStreetLine2of2'),
```

Comment:
It would be good to have consistent formatting for `1of2` format. These two are different:
- `line1Of2`
- `buildingAndStreetLine2of2`

I prefer the second option.

Reply:
Done.